specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json

```diff
@@ -9241,6 +9245,27 @@
       },
       "description": "The protection policy of a virtual machine scale set VM."
     },
+    "ScheduledEventsProfile": {
+      "properties": {
+        "terminateNotificationProfile":
+        {
+          "$ref":"#definitions/TerminateNotificationProfile",
+          "description": "Specifies Terminate Scheduled Event related configurations."
+        }
+      }
+    },
+    "TerminateNotificationProfile": {
+      "properties": {
+        "notBeforeTimeout": {
+          "type": "string",
+          "description" : "Configurable length of time a Virtual Machine being deleted will have to potentially approve the Terminate Scheduled Event before the event is auto approved (timed out). The configuration must be specified in ISO 8601 format, the default value is 5 minutes (PT5M)"
+        },
+        "enable" : {
```

Comment:
Is this optional?  What is the default value if this property is not given?

Reply:
The description I wrote says the default is 5 minutes

Reply:
I mean, the default value for the Boolean variable "enable".

Reply:
Ah default for enable is false